diff --git a/index.d.ts b/index.d.ts
index 5e7d6b88aefa96a184a29f2420779e50d3374835..ed5a30aab1046155e08ef3fe76bb519dae1b94a4 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -12,7 +12,7 @@ declare class Pbf {
     constructor(buffer?: Uint8Array | ArrayBuffer);
 
     destroy(): void;
-    readFields<T>(readField: (tag: number, result?: T, pbf?: Pbf) => void, result?: T, end?: number): T;
+    readFields<T>(readField: (tag: number, result: T, pbf: Pbf) => void, result?: T, end?: number): T;
     readMessage<T>(readField: (tag: number, result?: T, pbf?: Pbf) => void, result?: T): T;
     readFixed32(): number;
     readSFixed32(): number;
@@ -51,7 +51,7 @@ declare class Pbf {
     writeDouble(val: number): void;
     writeBytes(buffer: Uint8Array): void;
     writeRawMessage<T>(fn: (obj: T, pbf?: Pbf) => void, obj?: T): void;
-    writeMessage<T>(tag: number, fn: (obj: T, pbf?: Pbf) => void, obj?: T): void;
+    writeMessage<T>(tag: number, fn: (obj: T, pbf: Pbf) => void, obj?: T): void;
     writePackedVarint(tag: number, arr: number[]): void;
     writePackedSVarint(tag: number, arr: number[]): void;
     writePackedBoolean(tag: number, arr: boolean[]): void;
